### ISSUE TITLE
@types/node incompatible with @types/webpack-env

### ISSUE THREAD

**Issue by jlsnake** [DefinitelyTyped/DefinitelyTyped#29280]
- [x] I tried using the `@types/node@10.11.2` and `@types/webpack-env@1.13.6` package and had problems.
- [x] I tried using the latest stable version of tsc. https://www.npmjs.com/package/typescript
- [x] I have a question that is inappropriate for [StackOverflow](https://stackoverflow.com/).  (Please ask any appropriate questions there).
- [x] [Mention](https://github.com/blog/821-mention-somebody-they-re-notified) the authors (see `Definitions by:` in `index.d.ts`) so they can respond.
  - Authors: @jkomyno @a-tarasyuk @use-strict @rhonsby 

In the end of compile process I got some errors:
```
ERROR in [at-loader] ./node_modules/@types/node/index.d.ts:196:15
    TS2451: Cannot redeclare block-scoped variable 'process'.

ERROR in [at-loader] ./node_modules/@types/node/index.d.ts:243:15
    TS2451: Cannot redeclare block-scoped variable 'require'.

ERROR in [at-loader] ./node_modules/@types/node/index.d.ts:256:15
    TS2451: Cannot redeclare block-scoped variable 'module'.

ERROR in [at-loader] ./node_modules/@types/webpack-env/index.d.ts:203:13
    TS2451: Cannot redeclare block-scoped variable 'require'.

ERROR in [at-loader] ./node_modules/@types/webpack-env/index.d.ts:254:13
    TS2451: Cannot redeclare block-scoped variable 'module'.

ERROR in [at-loader] ./node_modules/@types/webpack-env/index.d.ts:262:13
    TS2451: Cannot redeclare block-scoped variable 'process'.
```

What should I do?

**Comment by actraiser** [comment]
I fixed this recently in #29267 - just upgrade to @types/node@10.11.3 and it will work again.

greets
-act

**Comment by jlsnake** [comment]
Thank you. It works.
